### PR TITLE
Spelling fixes from Brad McFall and ZainCypher and addition of DexalotSubnetBalances

### PR DESCRIPTION
**Pull request by ngurmen**

### CODE REVIEW

**Review by BradMcFall — CHANGES_REQUESTED** [review]
image
PortfolioBridgeSub.md
old(red) line 266 
new(green) line 267
should be changed 

from "to the portfoBrige. Portfo(lio)BrigeSub the list"
to   "to the portfolioBridge. PorfolioBridgeSub the list"

This is the only thing I see. Everything else looks good.

File: docs/contracts/PortfolioBridge.md [hunk 1]
```diff
@@ -263,13 +264,14 @@ function setGasForDestinationLzReceive(uint256 _gas) external
 
 #### addToken
 
-Adds the given token to the portfoBrige. PortfoBrigeSub the list will be bigger as they could be from
```
Comment (BradMcFall) [inline docs/contracts/PortfolioBridge.md:266]: Change PortfoBridgeSub -> PortfolioBridgeSub

File: docs/contracts/PortfolioBridge.md [hunk 2]
```diff
@@ -11,11 +11,11 @@ will be added as needed. This contract encapsulates all bridge provider implemen
 doesn&#x27;t need to know about.
 
 **Dev notes:** \
-The information flow for messages between PortfolioMain and PortfolopSub is as follows:
```
Comment (BradMcFall) [inline docs/contracts/PortfolioBridge.md:14]: Change PortfolopSub --> PorfolioSub

File: docs/contracts/PortfolioBridge.md (hunk 1 above)
Comment (ngurmen) [reply docs/contracts/PortfolioBridge.md:266]: done

File: docs/contracts/PortfolioBridge.md (hunk 2 above)
Comment (ngurmen) [reply docs/contracts/PortfolioBridge.md:14]: done

File: docs/contracts/bridgeApps/LzApp.md [hunk 3]
```diff
@@ -1,366 +1,382 @@
...
-| ---- | ---- | ----------- |
-| _srcChainId | uint16 | Source(Remote) chain id |
-| _path | bytes | abi.encodePacked(sourceAddress, localAddress) |
-
+
+| Name | Type | Description |
+| ---- | ---- | ----------- |
+| _srcChainId | uint16 | Source(Remote) chain id |
+| _path | bytes | Remote contract address concatenated with the local contract address, 40 bytes
 abi.encodePacked(sourceAddress, localAddress) |
```
Comment (BradMcFall) [inline docs/contracts/bridgeApps/LzApp.md:185]: is the ^M correct ?

File: docs/contracts/PortfolioBridge.md (hunk 1 above)
Comment (BradMcFall) [inline docs/contracts/PortfolioBridge.md:266]: One more change here - PortfoBrigeSub --> PortfolioBridgeSub

File: docs/contracts/bridgeApps/LzApp.md (hunk 3 above)
Comment (ngurmen) [reply docs/contracts/bridgeApps/LzApp.md]: corrected

File: docs/contracts/PortfolioBridge.md (hunk 1 above)
Comment (ngurmen) [reply docs/contracts/PortfolioBridge.md:266]: done

Comment (BradMcFall) [reply docs/contracts/PortfolioBridge.md:266]: Sorry but still one more "d" missing in line 267 the

**Review by BradMcFall — APPROVED** [review]

**Pull request merged**